buster/retriever/deeplake.py

```diff
@@ -58,20 +58,29 @@ def build_tql_query(embedding, sources=None, top_k: int = 3):
 
 
 class DeepLakeRetriever(Retriever):
-    def __init__(self, path, exec_option: str = "python", org_id: str = None, use_tql: bool = False, **kwargs):
+    def __init__(self, path, exec_option: str = "python", use_tql: bool = False, deep_memory=False, **kwargs):
         from deeplake.core.vectorstore import VectorStore
 
         super().__init__(**kwargs)
+        activeloop_token = os.getenv("ACTIVELOOP_TOKEN")
```

Comment:
Sorry I wasn't clear, I meant having the envvar as an argument of the init. This is not great having an env var being loaded deep within a library, it should be exposed as much as possible